src/wizard/java/logback.md

```diff
@@ -24,7 +24,7 @@ Install Sentry's integration with Logback using either Maven or Gradle:
 ### Gradle
 
 ```groovy
-implementation 'io.sentry:sentry-logback:3.2.0'
+implementation 'io.sentry:sentry-logback:3.2.1'
```

Comment:
we went away from the placeholder {version} and now the whole Java SDK deps have different versions, little of a mess.
Search for: `implementation 'io.sentry`

Reply:
the solution is using `craft` though, not making the user figure out the right version

Reply:
agreed, but now there's a pinned version in the docs that is not up to date and every page has a different version, not sure which one is worse.